profiling/energy-profiler/daemon.hpp

```diff
@@ -0,0 +1,39 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+
+#pragma once
```

Comment:
We should probably agree at the level of `kokkos-tools` on using consistently either preprocessor conditionals or this pragma to protect our includes.

Reply:
I agree that consistency would be a good thing to aim for. There are a lot more things in Tools that could be made consistent though.

Reply:
switched to classic ifdef in https://github.com/kokkos/kokkos-tools/pull/300/commits/67a80309e4e776863c24aebcf22b9bec5f023559